R/05_analysis_1.qmd

```diff
@@ -0,0 +1,82 @@
+
+```{r}
+    #| message: false
+    library(dplyr)
+    library("tidyverse")
+    rm(list = ls())
+
+
+diabetes_data <- read_tsv(file = "../data/04_diabetes_data_augment.tsv")
+```
+
+```{r}
+plot1 <- diabetes_data |>
+                  mutate(income = factor(income, levels = c('[0,5000)','[5000,10000)','[10000,15000)','[15000,20000)','[20000,25000)',
+                          '[25000,35000)','[35000,45000)','[45000,55000)','[55000,65000)','[65000,75000)','[75000,100000)'))) |>
+                  filter(!is.na(income)) |>
+                  group_by(income) |>
+                  summarize(percentage = sum(diagnosis == 1)*100 / n())
+
```

Comment:
filter(!is.na()) is Base R and not tidyverse.
Example of code you could use instead:
  diabetes_data |>
       select(income,
                diagnosis) |>
  drop_na() |>
  group_by(income) |>
  summarize(percentage = sum(diagnosis == 1)*100 / n())